app/slave/subjob_executor.py

```diff
@@ -2,7 +2,8 @@
 import shutil
 import time
 
-from app.master.subjob import Subjob
+from app.master.build import BuildArtifact
```

Comment:
Nitpicking (again). It's kind of weird to import packages from `master` for the slave service. How about moving BuildArtifact to someplace else?